js-webapp/README.md

```diff
@@ -187,30 +168,27 @@ this JavaScript library, we can find the root of the mathematical function, and
 following HTML:
 
 ```html
-<!doctype html>
-<html lang="en">
+<html>
    <head>
-      <title>Newton-Raphson</title>
-      <meta charset="utf-8">
       <script type="text/javascript" src="newtonraphson.js"></script>
+   </head>
+   <body>
+      <div id="answer"></div>
       <script>
-         createModule().then((module) => {
+         createModule().then((NewtonRaphson) => {
```

Comment:
I would use `rootfinding` instead as that is the C++ namespace in which the class is defined.
```suggestion
         createModule().then((rootfinding) => {
```

Reply:
done